### ISSUE TITLE
您好,我在解码时遇到了一些问题

### ISSUE THREAD

**Issue by Sssyz** [calmisential/YOLOv3_TensorFlow2#4]
您好,在运行时出现了这样的错误UnicodeDecodeError: 'utf-8' codec can't decode byte 0xd5 in position 95: invalid continuation byte

这里出现了问题 train_from_scratch.py, line 74, in <module>
train_step(image_batch=process_image_filenames(images), label_batch=labels)

**Comment by calmiLovesAI** [comment]
确保configuration.py中的参数是设置正确的，还有数据集都在正确的目录下。

**Comment by Sssyz** [comment]
谢谢!
我还有一个疑问，我在第一次迭代时Loss下降情况正常。
但在第二次迭代训练集时，loss会在到达一个数后（如84）开始慢慢变大又慢慢变小，这样一直反复，但不会继续下降，调低了learning_rate也没有改进，请问是什么问题?如何能让loss继续下降？
十分感谢!




------------------&nbsp;原始邮件&nbsp;------------------
发件人: "calmisential"<notifications@github.com&gt;; 
发送时间: 2020年1月21日(星期二) 上午9:16
收件人: "calmisential/YOLOv3_TensorFlow2"<YOLOv3_TensorFlow2@noreply.github.com&gt;; 
抄送: "宋昱"<784386351@qq.com&gt;; "Author"<author@noreply.github.com&gt;; 
主题: Re: [calmisential/YOLOv3_TensorFlow2] 您好,我在解码时遇到了一些问题 (#4)




确保configuration.py中的参数是设置正确的，还有数据集都在正确的目录下。

**Comment by calmisential** [comment]
> 
> 谢谢! 我还有一个疑问，我在第一次迭代时Loss下降情况正常。 但在第二次迭代训练集时，loss会在到达一个数后（如84）开始慢慢变大又慢慢变小，这样一直反复，但不会继续下降，调低了learning_rate也没有改进，请问是什么问题?如何能让loss继续下降？ 十分感谢!
> […](#)
> ------------------&nbsp;原始邮件&nbsp;------------------ 发件人: "calmisential"<notifications@github.com&gt;; 发送时间: 2020年1月21日(星期二) 上午9:16 收件人: "calmisential/YOLOv3_TensorFlow2"<YOLOv3_TensorFlow2@noreply.github.com&gt;; 抄送: "宋昱"<784386351@qq.com&gt;; "Author"<author@noreply.github.com&gt;; 主题: Re: [calmisential/YOLOv3_TensorFlow2] 您好,我在解码时遇到了一些问题 (#4) 确保configuration.py中的参数是设置正确的，还有数据集都在正确的目录下。 — You are receiving this because you authored the thread. Reply to this email directly, view it on GitHub, or unsubscribe.

你可以检查数据集的问题，有没有不好的数据，或者更换优化器，调整learning_rate衰减策略。

**Comment by ghost** [comment]
> 您好，在运行时出现了这样的错误UnicodeDecodeError：“ utf-8”编解码器无法解码位置95处的字节0xd5：无效的连续字节
> 
> 这里出现了问题train_from_scratch.py​​，第74行，位于
> train_step中（image_batch = process_image_filenames（图像），label_batch = labels）

您好 我遇到了和你一样的问题， 
Syntax Error: (unicode error) 'unicodeescape' codec can't decode bytes in position 166-167: truncated \uXXXX escape 可以确定数据集目录位置正确， 请问configuration.py中您修改了哪些位置？
我尝试修改了，也使用了默认的configuration 都不行
